torch_xla/csrc/ops/native_batch_norm_backward.cpp

```diff
@@ -34,14 +34,13 @@ xla::Shape NodeOutputShape(const Value& grad_out, const Value& input,
 NativeBatchNormBackward::NativeBatchNormBackward(
     const Value& grad_out, const Value& input, const Value& weight,
     const Value& save_mean, const Value& save_invstd, bool training, double eps)
-    : Node(
-          ir::OpKind(at::aten::native_batch_norm_backward),
-          {grad_out, input, weight, save_mean, save_invstd},
-          [&]() {
-            return NodeOutputShape(grad_out, input, weight, save_mean,
-                                   save_invstd, training);
-          },
-          /*num_outputs=*/3, xla::util::MHash(training, eps)),
+    : Node(ir::OpKind(at::aten::native_batch_norm_backward),
```

Comment:
We should not be using global reformat scripts as they tend to intermingle real changes with formatting changes.
Please drop those. I will fix them in a dedicated PR.